scipy/signal/ltisys.py

```diff
@@ -121,10 +121,10 @@ def tf2ss(num, den):
     if num.shape[-1] > 0:
         D = num[:, 0]
     else:
-        D = array([], float)
+        D = array([0], float)
```

Comment:
Probably a silly question: this change, can it interfere with other cases, when a transfer function is not trivial?